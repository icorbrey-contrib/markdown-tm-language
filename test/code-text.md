a `b` c.

More/less ```a``.

Less/more ``a```.

Less/More/Less ` `` `.

More/Less/More `` ` ``.

Stuff `a`a`.

Stuff \`a`a`.

Stuff \``a`a`.

a `b
c` d.
