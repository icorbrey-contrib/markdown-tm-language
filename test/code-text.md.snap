>a `b` c.
#^^ source.md meta.paragraph.md
#  ^ source.md meta.paragraph.md markup.code.other.md string.other.begin.code.md
#   ^ source.md meta.paragraph.md markup.code.other.md markup.raw.code.text.md
#    ^ source.md meta.paragraph.md markup.code.other.md string.other.end.code.md
#     ^^^^ source.md meta.paragraph.md
>
>More/less ```a``.
#^^^^^^^^^^^^^^^^^^ source.md meta.paragraph.md
>
>Less/more ``a```.
#^^^^^^^^^^^^^^^^^^ source.md meta.paragraph.md
>
>Less/More/Less ` `` `.
#^^^^^^^^^^^^^^^ source.md meta.paragraph.md
#               ^ source.md meta.paragraph.md markup.code.other.md string.other.begin.code.md
#                ^^^^ source.md meta.paragraph.md markup.code.other.md markup.raw.code.text.md
#                    ^ source.md meta.paragraph.md markup.code.other.md string.other.end.code.md
#                     ^^ source.md meta.paragraph.md
>
>More/Less/More `` ` ``.
#^^^^^^^^^^^^^^^ source.md meta.paragraph.md
#               ^^ source.md meta.paragraph.md markup.code.other.md string.other.begin.code.md
#                 ^^^ source.md meta.paragraph.md markup.code.other.md markup.raw.code.text.md
#                    ^^ source.md meta.paragraph.md markup.code.other.md string.other.end.code.md
#                      ^^ source.md meta.paragraph.md
>
>Stuff `a`a`.
#^^^^^^ source.md meta.paragraph.md
#      ^ source.md meta.paragraph.md markup.code.other.md string.other.begin.code.md
#       ^ source.md meta.paragraph.md markup.code.other.md markup.raw.code.text.md
#        ^ source.md meta.paragraph.md markup.code.other.md string.other.end.code.md
#         ^^^^ source.md meta.paragraph.md
>
>Stuff \`a`a`.
#^^^^^^ source.md meta.paragraph.md
#      ^^ source.md meta.paragraph.md constant.language.character-escape.md
#        ^ source.md meta.paragraph.md
#         ^ source.md meta.paragraph.md markup.code.other.md string.other.begin.code.md
#          ^ source.md meta.paragraph.md markup.code.other.md markup.raw.code.text.md
#           ^ source.md meta.paragraph.md markup.code.other.md string.other.end.code.md
#            ^^ source.md meta.paragraph.md
>
>Stuff \``a`a`.
#^^^^^^ source.md meta.paragraph.md
#      ^^ source.md meta.paragraph.md constant.language.character-escape.md
#        ^^ source.md meta.paragraph.md
#          ^ source.md meta.paragraph.md markup.code.other.md string.other.begin.code.md
#           ^ source.md meta.paragraph.md markup.code.other.md markup.raw.code.text.md
#            ^ source.md meta.paragraph.md markup.code.other.md string.other.end.code.md
#             ^^ source.md meta.paragraph.md
>
>a `b
#^^^^^ source.md meta.paragraph.md
>c` d.
#^^^^^^ source.md meta.paragraph.md
>